documentation/faq.mdx

```diff
@@ -100,14 +92,11 @@ The original game had a few issues of varying severity - from progress softlocks
 
 ### Will there be mod support?
 
-In the future yes! We intend to make it as easy as we can to make your own code or import graphics for the game. In the meanwhile several mods can already be tried out using the unofficial mod launcher, found at 
+In the future yes! We intend to make it as easy as we can to make your own code or import graphics for the game. In the meanwhile several mods can already be tried out using the unofficial mod launcher, found at https://jakmods.dev/
 
 ### Will you support pressure sensitivity?
 
-No. Pressure sensitivity is the possibility of a few controllers such as DualShock2, DualShock3 to give analog states as opposed to not binary states (pressed or not pressed) for face buttons. Even though this seems appealing for driving vehicles, there is no plan to support this, since it requires a massive game rewrite. On top of that, the vast majority of controllers doesn't support this at a hardware or driver level anyway.
-
-We do intend to look into supporting adaptive triggers in the future, as soon as third party libraries officially support this.
-<!-- https://discord.com/channels/756287461377703987/1228112614077235200 -->
+No. Pressure sensitivity is the possibility of a few controllers such as DualShock 2, DualShock 3 to detect differences in the amount of force applied to them, as opposed to binary inputs (e.g. "pressed" or "not pressed"). Even though pressure sensitivity is present in the PS2 and PS3 versions (for extra jump height and controlling the zoomer), there is no plan to support this for face buttons. The vast majority of controllers doesn't support this at a hardware or driver level anyway. In the future, we possibly support an option to use triggers to accelerate combined with pressure sensitivity.
```

Comment:
"for face buttons" should be removed, since pressure sensitivity works on all 12 buttons on the DS2 excluding sticks and start/select.

Reply:
Apologies, this was still written with a misunderstanding, apparently. I will change that and ask for a review right away.

Reply:
Hm, @xTVaser made a note on it that people want to use triggers (that is, R2, to my understanding) and using R2 would feel foreign. When editing the commit, I noticed I also had this part in there. Should I omit it?

> In the future, we possibly support an option to use triggers to accelerate combined with pressure sensitivity.

Apologies for the many miscommunications so far.